### DISCUSSION TITLE
https://degen.tips/

### DISCUSSION THREAD

**Discussion by soheltaj7** [blockscout/blockscout#9972] [Support] [answered]
This Degen tips airdrop is real or fack??
![IMG_20240429_115443](https://github.com/blockscout/blockscout/assets/36280079/8dca6351-4b0d-4728-b2a4-26e8e4098138)
![IMG_20240429_115411](https://github.com/blockscout/blockscout/assets/36280079/35199b73-0cff-453c-9aeb-a11543623111)

Airdrop 2
In Season 3, you can earn points on Farcaster by receiving tips from others. You'll have a daily tip allowance, visible on this page once you connect your wallet. To leave a tip, comment under a cast with "amount $DEGEN", for example, "420 $DEGEN". If you exceed your daily limit, additional tips won't count. You can tip on multiple casts each day. Tips in specific channels get a boost. Control who receives Degen – happy tipping!

Users found farming

**Comment by ekawibisono** [comment, marked as answer]
Hello @soheltaj7 This is blockscout github, please contact degen team, thanks
